.travis.yml

```diff
@@ -12,63 +12,73 @@ php:
 # Set up environment variables.
 env:
   global:
-    # add composer's global bin directory to the path
-    # see: https://github.com/drush-ops/drush#install---composer
+    # Global
     - PATH="$PATH:$HOME/.composer/vendor/bin"
 
-    # Configuration variables.
+    # Composer
+    - COMPOSER_EXIT_ON_PATCH_FAILURE=1
+
+    # DRUPAL_TI Config.
     - DRUPAL_TI_MODULE_NAME="contacts"
-    #- DRUPAL_TI_SIMPLETEST_GROUP="user"
-    - DRUPAL_TI_PULL_SIMPLETEST_GROUP="contacts"
-    - DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="contacts"
-    #- DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="user,contacts"
-    #- DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="--all"
+    - DRUPAL_TI_SIMPLETEST_GROUP="contacts"
 
-    # The environment to use, supported are: drupal-7, drupal-8
+    # DRUPAL_TI Env
     - DRUPAL_TI_ENVIRONMENT="drupal-8"
     - DRUPAL_TI_CORE_BRANCH="8.2.x"
```

Comment:
Are we able to take the latest stable or take it from our dependency info? I feel like this is going to get forgotten!